### ISSUE TITLE
BERT Sequence Classification: Accuracy Fluctuations & Inefficient Learning

### ISSUE THREAD

**Issue by bkonkle** [bkonkle/burn-transformers#6]
This ticket will carry the details of my difficulty implementing sequence classification using the pretrained weights from `bert-base-uncased` using the BERT implementation from the `models` repo.

I believe the architecture is correct, but I'm not seeing the kind of learning I expect compared to the PyTorch implementation.

_More detail to come..._
